applications/AdjointFluidApplication/custom_elements/vms_adjoint_element.h

```diff
@@ -267,6 +267,17 @@ class VMSAdjointElement: public Element {
         KRATOS_CATCH("")
     }
 
+    void CalculateLeftHandSide(MatrixType& rLeftHandSideMatrix,
+                               ProcessInfo& /*rCurrentProcessInfo*/) override
+    {
+        if (rLeftHandSideMatrix.size1() != TFluidLocalSize || rLeftHandSideMatrix.size2() != TFluidLocalSize)
+            rLeftHandSideMatrix.resize(TFluidLocalSize,TFluidLocalSize,false);
+
+        for (IndexType i=0; i < TFluidLocalSize; i++)
```

Comment:
just to tell that you could do this shortly as

     rLeftHandSideMatrix.clear();